log/wrapper.go

```diff
@@ -289,7 +289,7 @@ type Counter interface {
 // baseplate.ParseConfigYAML call, for example:
 //
 //	// a global variable
-//	var tracingFailures = promauto.NewCounter(prometheus.CounterOpts{
+//	var tracingFailures = promauto.With(internalv2compat.GlobalRegistry).NewCounter(prometheus.CounterOpts{
```

Comment:
this example certainly should not add that :)

Reply:
whoops, I did a find/replace and missed a few that shouldn't be covered, thanks